src/Compiler/CompilerUtils.cpp

```diff
@@ -327,6 +333,9 @@ std::string getTargetFilename(
 // Returns 0 on success, error code on failure.
 static int genLLVMBitcode(const mlir::OwningOpRef<ModuleOp> &module,
     std::string outputNameNoExt, std::string optimizedBitcodeNameWithExt) {
+  auto llvmScope =
+      rootScope.nest("[onnx-mlir] Compiling to LLVM Optimized Bitcode");
```

Comment:
`[onnx-mlir] Compiling to LLVM Optimized Bitcode"` => `[onnx-mlir] Compiling MLIR module to LLVM Optimized Bitcode"`